app/models/squix/print_job.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+module Squix
+  # labels:
```

Comment:
This is where we will need to put a bit of thought into how we structure the labels ...